user_verification/application/services/user_verification_service.py

```diff
@@ -0,0 +1,46 @@
+import requests
+
+from common.constant import StatusCode
+from user_verification.config import JUMIO_BASE_URL, JUMIO_API_KEY
+from user_verification.infrastructure.repositories.user_verification_repository import UserVerificationRepository
+
+user_verification_repo = UserVerificationRepository()
+
+
+class UserVerificationService:
+    def __init__(self):
+        self.request_headers = {
+            "accept": "application/json",
+            "content-type": "application/json",
+            "content-length": 3495,
+            "authorization": f"Basic {JUMIO_API_KEY}"
+        }
+
+    def initiate(self, payload):
```

Comment:
In initiate generate unique verification id with pending status, pass that id to third party verification and expect that id in callback to update verification status in our database